index.html

```diff
@@ -55,6 +56,19 @@ <h2 class="font-light">Relationship anarchy, as radiant as the sun.</h2>
         </div>
       </div>
     </div>
+    <div class="container">
+      <div class="row">
+        <div class="col-12 center">
+          <h2 id="what-is-this">What is this?</h2>
+          <p>This is a tool to help you discuss and explore the relationship you'd like with your loved ones. It's inspired by the <a href="//www.multiamory.com/podcast/339-the-smorgasbord-of-relationships">Relationship Anarchy Smorgasbord</a>, which was initially developed by Lyrica Lawrence and Heather Orr in 2016.</p>
+          <h2 id="what-is-this">How do I use it?</h2>
```

Comment:
`id`s must be unique in html

Reply:
oops, copy paste fail, thanks for catching it!